include/swift/SIL/SILInstruction.h

```diff
@@ -323,13 +323,17 @@ class SILInstruction : public llvm::ilist_node<SILInstruction> {
 
   /// A backreference to the containing basic block.  This is maintained by
   /// ilist_traits<SILInstruction>.
-  SILBasicBlock *ParentBB;
+  SILBasicBlock *ParentBB = nullptr;
 
-  /// This instruction's containing lexical scope and source location
-  /// used for debug info and diagnostics.
-  SILDebugLocation Location;
+  /// This instruction's containing lexical scope for debug info.
+  const SILDebugScope *debugScope = nullptr;
```

Comment:
Nit: are we supposed to be following the LLVM coding style with upper-case members in this file (`ParentBB` vs. `debugScope`)? I don't really care but it might be nice to be consistent...

Reply:
Well, for new code we usually use lower-case for variables/members. I'll try to make it more consistent.